Cargo.toml

```diff
@@ -1,10 +1,21 @@
+[workspace]
+members = [".", "scripts"]
+
 [package]
 name = "commonware-avs-router"
 version = "0.1.0"
 edition = "2024"
 
 [dependencies]
-alloy = { version = "0.12.6" }
+# Workspace dependencies
+alloy = { workspace = true }
+tokio = { workspace = true }
```

Comment:
this might cause future compat issues with commonware

Reply:
can you explain that a bit more?